src/pages/api/save-response.ts

```diff
@@ -0,0 +1,27 @@
+import type { NextApiRequest, NextApiResponse } from 'next';
+import { initializeFirestore } from '../../utils/get-firestore';
+import { doc, setDoc } from 'firebase/firestore/lite';
+
+type Data = {
+  id: number;
+  answers: string[];
+};
+
+export default async function saveResponses(req: NextApiRequest, res: NextApiResponse<Data>) {
+  const db = await initializeFirestore();
+
+  if (req.method === 'GET') {
+    res.status(200).json({ id: Date.now(), answers: ['23', 'get responses was called'] });
+    console.log('get responses was called');
```

Comment:
I don't think it serves any purpose to handle `GET` or any other method here besides `POST` since this endpoint doesn't support them anyway. If the method is anything other than `POST` just return a 405 (https://en.wikipedia.org/wiki/List_of_HTTP_status_codes#4xx_client_errors) code.